src/theme/index.ts

```diff
@@ -43,3 +45,10 @@ export const RootTheme = {
 
 // Exporting out @xstyled's ThemeProvider. Only way I could get it to work w/o issues.
 export class ThemeProvider extends XstyledThemeProvider {}
+
+// Passing RootTheme to avoid circular references with exports. RootTheme is the default baseTheme
+// but the user can pass in any theme.
+export const themeMerge = (
```

Comment:
Is there still a need to do this and export from the theme file, as opposed to exporting the function directly from the utils file

Reply:
None of the util functions are exports for the main Anchor build. If we want to start doing that, we can. I have no strong opinion.